src/auth/constants.ts

```diff
@@ -1,3 +1,3 @@
 export const jwtConstants = {
-  secret: 'SEGREDO_SUPER_SECRETO'
-};
\ No newline at end of file
+  secret: 'SEGREDO_SUPER_SECRETO',
```

Comment:
Isso aqui tem que vir do .env, porque senão você sobe o segredo do JWT pro repositório. E ele é inclusive mais importante do que a senha de um cliente, porque com ele um atacante pode se passar por qualquer usuário.
Algo assim:
```suggestion
  secret: process.env.JWT_SECRET,
```

E adicionar no .env.example

Reply:
Atualizado ✅